src/screens/settings/css/profile.css

```diff
@@ -129,21 +151,57 @@ font-family: Arial, Helvetica, sans-serif;
 /* Next Line of code, Display Part */
 
 .cont3{
-height: 439.19px;
+height: 250vh;
```

Comment:
vh is related to % so can be 100vh at most, in relation to the screen

Reply:
Okay